rlkit/samplers/util.py

```diff
@@ -51,6 +52,8 @@ def rollout(env, agent, max_path_length=np.inf, animated=False, is_online=False)
         o = next_o
         if animated:
             env.render()
+    if resample == 'trajectory':
```

Comment:
personally don't think this should belong in the rollout method (should just be called in the while loop in obtain_samples)

Reply:
I disagree because I think all the resampling should happen in the same function, and transition-level resampling has to happen here. Though if we generalize it to resample after any number of transitions, we'll run into this problem again...

Reply:
My reasoning on keeping it out is that rollout shouldn't care about what happens after the rollout, but I'm fine either way. I'm against resampling after arbitrary numbers of transitions. I think it should just be sample whole trajectories until we reach a threshold to resample.

Reply:
ok I think we should just sample by trajectory at test time to keep things consistent, so I'll move this out of the rollout function

Reply:
see `da9217c`